### PR TITLE
Exporter - history sync

### PR DESCRIPTION
**Pull request by amir-blox**
Introduce history sync - not enabled by default (yet)

### CODE REVIEW

File: exporter/ibft/reader.go [hunk 1]
```diff
@@ -68,35 +67,41 @@ func (r *reader) Sync() error {
...
+	return err
 }
 
 // Start starts the network listeners
 func (r *reader) Start() {
-	r.listenToNetworkDecidedMessages()
+	//r.listenToNetworkDecidedMessages()
 }
 
-// listenToNetworkDecidedMessages listens for decided messages
```
Comment (alonmuroch) [inline exporter/ibft/reader.go:80]: @amir-blox should this be commented out?

File: exporter/ibft/reader.go [hunk 2]
```diff
@@ -105,53 +110,53 @@ func (r *reader) validateDecidedMsg(msg *proto.SignedMessage) error {
...
-	highest, err := r.storage.GetHighestDecidedInstance(msg.Message.Lambda)
-	if err != nil {
-		if err.Error() == kv.EntryNotFoundError {
-			return msg.Message.SeqNumber > 0, nil
-		}
-		return false, errors.Wrap(err, "could not get highest decided instance from storage")
-	}
-	return highest.Message.SeqNumber < msg.Message.SeqNumber, nil
-}
+//// processDecidedMessage is responsible for processing an incoming decided message.
```
Comment (alonmuroch) [inline exporter/ibft/reader.go:113]: @amir-blox should this be commented out?

**Review by alonmuroch — CHANGES_REQUESTED** [review]

File: exporter/ibft/reader.go (hunk 1 above)
Comment (amir-blox) [reply exporter/ibft/reader.go]: It was removed due to irrelevancy

File: exporter/ibft/reader.go (hunk 2 above)
Comment (amir-blox) [reply exporter/ibft/reader.go]: It was removed due to irrelevancy

**Pull request merged**